index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">  
+  <header>
+    <div class="top-header"><img src="images/Time.png" alt="time">
+    <img src="images/wifi.png" alt="wifi">
+  </div>
+    <div class="bottom-header">
+      <h1>Welcome</h1>
+      <span class="identical-to">&#8801;</span>
+  </div>
+  </header> 
+  <main>
+    <div class="intro">
+    <h2 class="first-name">Hey there. I'm Stephen</h2>
+    <h2 class="first-name software-dev">I'm a software developer</h2>
```

Comment:
- [ ] The Html file structure stipulates that we use an `h1` tag for the first header of our page, then we can use `h2`, `h3` tags, and so on for the following headers after the `h1` tag.
This practice helps make our HTML files more readable to browsers, kindly change the `h2` tags to `h1` tags.

Reply:
You are right, thanks!